python/src/uagents/resolver.py

```diff
@@ -281,9 +281,9 @@ async def _api_resolve(self, destination: str) -> Tuple[Optional[str], List[str]
 
             if response.status_code != 200:
                 if response.status_code != 404:
-                    LOGGER.warning(
-                        f"Failed to resolve agent {address} from {self._almanac_api_url},"
-                        f"querying Almanac contract..."
+                    LOGGER.debug(
```

Comment:
As long as the agent endpoint gets resolved somehow, I think this shouldn't rise to the WARNING level.